core/modules/url_filter.h

```diff
@@ -21,6 +24,9 @@ class Flow {
   uint32_t dst_ip;
   uint16_t src_port;
   uint16_t dst_port;
+  uint32_t padding;
+
+  Flow() : padding(0) {}
```

Comment:
You're right, I think I was assuming value initialization (braces) and it's probably not being used that way elsewhere in the code.  So I was confused.

Perhaps we can settle on option 2) in general -- all classes should have constructors that explicitly list every variable in the initialize list -- either with a value or just () depending on whether the default is ok.